docs/src/concrete-playback.md

```diff
@@ -0,0 +1,59 @@
+# Concretely playing back proof harnesses
+
+When the result of a certain check comes back as a `FAILURE`,
+this _experimental_ feature allows users to debug their proof harness and understand what
+caused the failing check.
+
+## Setup
+
+The user needs to have a fairly recent version of the Kani source code somewhere on their computer.
+To do this, run `git clone https://github.com/model-checking/kani.git`.
+Then, the user needs to add the following lines to their `Cargo.toml` file:
+```toml
+[dev-dependencies]
+kani = { path = "{path_to_kani}/library/kani", features = ["exe_trace"] }
+```
+
+## Usage
+
+Run Kani with the `--gen-exe-trace` flag.
+After verifying the proof harness checks, Kani will extract concrete values for the `kani::any()` variables
+and generate a Rust unit test case.
+This unit test initializes the concrete values and calls the proof harness.
+A user can then either 1) copy this unit test into the same module as their proof harness or
+2) run Kani with the `--add-exe-trace-to-src` flag to have Kani automatically do this.
```

Comment:
After you talk about the testcase structure, I would introduce the new flag. Something like:

> Kani also offer an option to add the test directly to your source code. For that, invoke kani using the `--in-place` flag....

Reply:
done